### PR TITLE
Adding inline argument to PowerShell script

### PR DESCRIPTION
**Pull request by jalajmsft**

### CODE REVIEW

File: Tasks/AzureCLIV2/src/Utility.ts [hunk 1]
```diff
@@ -14,9 +14,10 @@ export class Utility {
             }
             throw new Error(tl.loc('JS_InvalidFilePath', filePath));
         }
-        let tmpDir = tl.getVariable('Agent.TempDirectory') || os.tmpdir();
+        let tempDirectory = tl.getVariable('Agent.TempDirectory') || os.tmpdir();
+        tl.checkPath(tempDirectory, `${tempDirectory} (agent.tempDirectory)`);
```
Comment (bishal-pdMSFT) [inline Tasks/AzureCLIV2/src/Utility.ts]: why is this checkPath required?

File: Tasks/AzureCLIV2/src/Utility.ts [hunk 2]
```diff
@@ -32,25 +33,29 @@ export class Utility {
...
+
         let contents: string[] = [];
         contents.push(`$ErrorActionPreference = '${powerShellErrorActionPreference}'`);
+        let filePath: string = tl.getPathInput("scriptPath", false, true);;
+        if (scriptLocation === "inlineScript") {
+            let inlineScript: string = tl.getInput("inlineScript", true);
+            filePath = path.join(tempDirectory, `azureclitaskscript${new Date().getTime()}_inlinescript.${fileExtensions[0]}`);
+            await Utility.createFile(filePath, inlineScript);
+        }
```
Comment (bishal-pdMSFT) [inline Tasks/AzureCLIV2/src/Utility.ts]:
```suggestion
        } else {
            if (!Utility.checkIfFileExists(filePath, fileExtensions)) {
               throw new Error(tl.loc('JS_InvalidFilePath', filePath));
            }
        }
```

File: Tasks/AzureCLIV2/src/Utility.ts [hunk 3]
```diff
@@ -32,25 +33,29 @@ export class Utility {
                 throw new Error(tl.loc('JS_InvalidErrorActionPreference', powerShellErrorActionPreference));
         }
 
+        // Write the script to disk.
+        tl.assertAgent('2.115.0');
+        let tempDirectory = tl.getVariable('Agent.TempDirectory') || os.tmpdir();
+        tl.checkPath(tempDirectory, `${tempDirectory} (agent.tempDirectory)`);
```
Comment (bishal-pdMSFT) [inline Tasks/AzureCLIV2/src/Utility.ts]: this checkPath can be removed as well

File: Tasks/AzureCLIV2/src/Utility.ts (hunk 1 above)
Comment (jalajmsft) [reply Tasks/AzureCLIV2/src/Utility.ts]: removed.

**Pull request merged**